Replace Vestine-derivatives in plant mutations, change uplink prices & hypopen to reflect changes

SlamBamActionman opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->

This PR does the following:

- Nocturine, Lexorin and Hyperzine has been removed from the botany random chem table.
- Impedrezene, Fresium, Heartbreaker, Benzene, Tear Gas, Bleach, Mannitol, Lipolicide, Wine, Beer, Coffee and Space Glue have been added to the botany chem table.
- Chemical Synthesis Kit has been reduced from 4 TC to 3 TC.
- Nocturine has been reduced from 6 TC to 4 TC.
- Hypopen has had its refill time per 5u reduced from 1.25s to 0.75s.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

As Vestine-derivatives, these chems are meant to be powerful and only available to antagonists. This allows them to be very potent and puts a clear line of power between crew and antag strength. 

Botany of course bypasses this, making balancing of these chems particularly difficult since one has to account for the possibility that the chems that are supposed to be among the strongest in the game become available to crew, or to antagonists in effectively infinite volumes. 

The chems replacing the vestine-derivatives are either precursor chems (meaning vestine only needs to be mixed in), unique chems difficult to procure, or chems that bring some specific effect.

Since Botany can no longer be a natural source of vestine-derivatives, the Chemical Synthesis Kit has been lowered to 3 TC. This should make it a more reasonable investment for Botanists to purchase who want to use top-of-the-line Syndie chems, as well as lower the barrier for Chemists and other crew who want to use Vestine chems. 

Nocturine has been lowered in price too, as after its nerf it no longer justifies the high price tag for antags who want to skip the chemical process. 

Since many of the strongest chems are now going to only be available in limited quantities, along with the chem changes introduced in #41693, I've also lowered the refill for the hypopen from 1.25 to 0.75 seconds. 1.25s felt too sluggish for even casual use, and 0.75s makes it feel slicker while still preventing active combat refills as was desired. 

---

Some things I am open to seeing implemented after this PR:

- The ability to make Omnizine using Vestine, to keep it from being a purely Botany-kept chem for antagonists. 
- The ability to grow Vestine-derivative chems for Botany by spiking trays with Vestine.
- An additional vestine vial in the Chemical Synthesis Kit, if the current amount ends up being too low for practical use in playtests.

## Technical details
<!-- Summary of code changes for easier review. -->

yaml

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

im not growing plants to show those changes

Hypopen changes:

https://github.com/user-attachments/assets/fbe5f360-d37c-4963-905b-eb43f94361bd

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Added Impedrezene, Fresium, Heartbreaker, Benzene, Tear Gas, Bleach, Mannitol, Lipolicide, Wine, Beer, Coffee and Space Glue to random plant reagent mutations.
- remove: Removed Nocturine, Lexorin and Hyperzine from random plant reagent mutations.
- tweak: Lowered Chemical Synthesis Kit uplink cost from 4 TC to 3 TC.
- tweak: Lowered Nocturine Bottle's uplink cost from 6 TC to 4 TC.
- tweak: Lowered the hypopen refill delay from 1.25s to 0.75s.

Princess-Cheeseballs commented:
Not sure how I feel about Caninase and Felinase being in there considering their explicit purpose was to be fun projects for chem to do. 

I'd rather not have botany creeping hugely into chem's domain.

SlamBamActionman commented:
> Not sure how I feel about Caninase and Felinase being in there considering their explicit purpose was to be fun projects for chem to do.

I am ambivalent on their inclusion, though between Caninaise, Felinaise, ~~Mayonnaise~~ and Corgium, Botany is lacking in some rare hightier fun chems.

Hitlinemoss commented:
> Not sure how I feel about Caninase and Felinase being in there considering their explicit purpose was to be fun projects for chem to do.
> 
> I'd rather not have botany creeping hugely into chem's domain.

I agree with this, and would also like to add that Bacchus' Blessing probably shouldn't be in the list either since that's a fun project for the Bartender to do. It's a semi-complicated recipe (with the hooch specifically requiring pure ethanol and welding fuel) so it would kind of undercut things if botany can just mass-produce the stuff.

Might be fun to have "simpler" alcoholic drinks (like wine, beer, etc) be in the list instead, for a few reasons:
- Doesn't creep too far into the Bartender's domain
- Still allows for fun botany projects ("grapes that get you drunk")
- Encourages cooperation with the Bartender by letting botany supply them with drink ingredients other than just, like, fruit juices

SlamBamActionman commented:
I agree with all the above comments.

- Removed Caninase, Felinase and Bacchus.
- Added Happiness, Mannitol, Lipolicide, Wine, Beer and Coffee.

Happiness *is* a precursor chem for Caninase and Corgium, but not the only one, and is already achievable in Botany to some degree via Rainbow Cannabis. It's in a rare randomness tier and the recipes require other exotic ingredients anyways so I don't think this impacts Chemistry too much.

Mannitol is a precursor for Psicodine but that also requires Impedrezene and some other stuff so I don't think it's much risk it'll invalidate the Chemistry playtime. It also has a very funny message when consumed ("You feel ENLIGHTENED") and that sounds like it'd make for some good eatin'.

Lipolicide is a Rainbow Cannabis reagent and as far as I can tell isn't used for anything but to make you immensely hungry.

SlamBamActionman commented:
> if you remove those rewarding chems you should replace it with something usefull that still works for antags and normal botany and not bloat

Impedrezene, Fresium, Heartbreaker, Benzene and Tear Gas are either precursor vestine chems (Nocturine and Lexorin) or have properties largely not seen elsewhere (Benzene is one of the very few reagents that deal Cellular damage). 

As has been expressed by others above, it's better if Botany is given specialized chems and mechanics in their stead, but that is a larger effort than this PR.

Princess-Cheeseballs reviewed (approved):
I approve of this PR but I'm holding it hostage until my other evil botany PR is approved because I too am evil.
